baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -158,13 +202,21 @@ def handle(self, message: confluent_kafka.Message) -> None:
 
                 context.metrics.gauge(f"{self.name}.{topic}.offset.{partition}").replace(offset)
         except Exception:
+            prom_success = "false"
```

Comment:
🔕  I didn't make a big deal out of this elsewhere but it'd be great if this was an actual boolean and converted to a lower case string when actually consumed